### PR TITLE
V2.2.0 - Fix for blocking calls

### PR DESCRIPTION
**Pull request by CSMastermind**

### CODE REVIEW

File: lib/session/establishContinuedServerSession.js [hunk 1]
```diff
@@ -60,8 +61,11 @@ function establishContinuedServerSession(req, controllerPath, initObjectTemplate
         }
     }
 
+
```
Comment (soitof) [inline lib/session/establishContinuedServerSession.js]: Extra line is extra.

**Comment by soitof** [comment]
Still approve it though.

File: lib/session/establishContinuedServerSession.js (hunk 1 above)
Comment (CSMastermind) [reply lib/session/establishContinuedServerSession.js]: Removed.

**Comment by soitof** [comment]
:shipit:

**Pull request merged**